python/pyspark/sql/tests/streaming/test_streaming_listener.py

```diff
@@ -304,6 +192,120 @@ def check_sink_progress(self, progress):
         self.assertTrue(isinstance(progress.numOutputRows, int))
         self.assertTrue(isinstance(progress.metrics, dict))
 
+
+class StreamingListenerTests(StreamingListenerTestsMixin, ReusedSQLTestCase):
```

Comment:
This test is refactored. I only keep the check_xx_evenet method in the Mixin class. Because of the behavior change, previous checks won't work in Connect.